### DISCUSSION TITLE
在网页上如何透过deep link链接到imtoken发起转帐?

### DISCUSSION THREAD

**Discussion by xxxddd12xd** [consenlabs/webview#75] [General]
![image](https://user-images.githubusercontent.com/34617152/212239213-8ce6d757-cf2d-49fb-bad9-cbdaec887504.png)
文档只给到这里,没给如何跳转至转帐发起页
顺带一提,我目前使用的链是tron,希望能透过USDT进行转帐
用tronWeb的话用户需要有chrome插件才能运行...

**Comment by xwartz** [comment]
没法通过 deeplink 直接发起 TRON 的转账，但是你可以通过 deeplink 打开 imToken，并且打开 DApp，然后在 DApp 里发起转账。

```
imtokenv2://navigate/DappView?url=https://fee.token.im/eth
```

参考：https://imtoken.gitbook.io/developers/products/deep-linking

**Reply by xxxddd12xd** [reply]
知道了,thx
